### PR TITLE
Classified Files ;)

### PR DESCRIPTION
**Pull request by FaisalKAlqahtani**
No comment

### CODE REVIEW

File: src/homeworkSolution.kt [hunk 1]
```diff
@@ -1,3 +1,55 @@
 fun main(){
 
+    //---------------------------------------------------------//
+    //Part1//
+    fun getLength(str: String?) : Int {
+
+        //First we create a variable and assign null to them
+        var checkNull : String? = null
+        //Then, use an if statement to determine the nullabilty of str
+        if(!checkNull.isNullOrEmpty()){
```
Comment (anasalqassim) [inline src/homeworkSolution.kt]: wow  you have a great idea here but I have one comment in this function you have checked the "str" if it is null or not 
you created a var that will always be null "checkNull" so you have to check if the "str " is null or empty 
what do you think can you do it?

**Comment by anasalqassim** [comment]
you done great job can you check my comment on your cod

File: src/homeworkSolution.kt (hunk 1 above)
Comment (FaisalKAlqahtani) [reply src/homeworkSolution.kt]: I was trying something new but i'll remove checkNull and try again

Comment (FaisalKAlqahtani) [reply src/homeworkSolution.kt]: The improved code is up!!

**Comment by anasalqassim** [comment]
great job thanks

**Pull request closed without merging**